AGENTS.md

```diff
@@ -0,0 +1,23 @@
+# AGENTS.md - GoDaddy JavaScript Style Guide
```

Comment:
This should just be GoDaddy JavaScript since it's no longer including components that are just for linting and formatting.